### PR TITLE
fix(@desktop/onboarding): not remove seed phrases when changing number of words

### LINKED ISSUE #5613: [Onboarding] Seed phrase input page changing between 12/18/24 words count shouldn't remove words from fields
### Description
Currently changing tab between 12/18/24 words, removes the previous introduced words. They should be kept.

### PR DESCRIPTION
**Pull request by endulab**
Fix #5613

### What does the PR do

The page remembers seed phrases when switching to different seed phrase modes.

### Affected areas

Onboarding: Add existing user/ Enter seed phrase

### Screenshot of functionality

https://user-images.githubusercontent.com/61889657/169807592-91b323b5-0b33-46d2-90ed-5eae18ffbb87.mov

### CODE REVIEW

**Comment by endulab** [comment]
@alexandraB99 can you test it ?

**Comment by alexandraB99** [comment]
> @alexandraB99 can you test it ?

@endulab looks good, however (I am not sure), I understood from the task that we should keep the words separately for each tab? that said, if for example I type a word in field 24 and switch to 18 or 12 word count, what would happen to that word?

**Comment by anastasiyaig** [comment]
@endulab was it planned to keep the data across the screens? For some reason i was expecting that app "knows" the words on particular screen (form) so when i switch from 12 to 24, it shows me empty form for 24 words and keeps whatever i left in 12 words form but it is a matter of preference for sure. I wonder if this is posted somewhere in figma or discussed to be done this way?


https://user-images.githubusercontent.com/82375995/169970139-7f1ce0d6-3e5b-4a7f-b39c-761993748875.mov

**Comment by anastasiyaig** [comment]
@endulab also, please make this happen for restoring the seed in wallet

1. enable wallet
2. open wallet
3. add account
4. advanced
5. import seed phrase

Screenshot 2022-05-24 at 11 09 09

**Comment by endulab** [comment]
@anastasiyaig @alexandraB99 the task was taken from Dubai-improvements list and it does not contain any more detailed description. For me it is a convenience for the user, eg. if he fills a lot of words and then decides to change the number of seed phrases, he does not need to fill them again. I am not sure why we need to keep words separately for each screen. But maybe I am wrong. @John-44 can help us to decide.

@anastasiyaig will check wallet.

**Comment by John-44** [comment]
@endulab I don't think we should keep words separately for each tab, I don't see any user need that keeping words separately would fulfil.

If a user is on 12 words and they switch to 18 words, the words they had already filled out on the 12 word screen should still be visible.  If a user is on the 18 word screen and they switch to the 12 word screen, the first 12 words from the 18 word screen should still be visible.

**Comment by alexandraB99** [comment]
> @endulab I don't think we should keep words separately for each tab, I don't see any user need that keeping words separately would fulfil.
> 
> If a user is on 12 words and they switch to 18 words, the words they had already filled out on the 12 word screen should still be visible. If a user is on the 18 word screen and they switch to the 12 word screen, the first 12 words from the 18 word screen should still be visible.

@John-44 "If a user is on the 18 word screen and they switch to the 12 word screen, the first 12 words from the 18 word screen should still be visible." -> if the user switches back to 18, should they see all 18 words or just 12?

File: ui/app/AppLayouts/Wallet/panels/ImportSeedPhrasePanel.qml [hunk 1]
```diff
@@ -162,7 +172,6 @@ GridView {
         width: _internal.seedPhraseInputWidth
         height: _internal.seedPhraseInputHeight
         textEdit.errorMessageCmp.visible: false
-        textEdit.input.anchors.topMargin: 11
```
Comment (endulab) [inline ui/app/AppLayouts/Wallet/panels/ImportSeedPhrasePanel.qml:165]: This fixes seed phrases visibility.

**Comment by endulab** [comment]
@anastasiyaig I added the feature to the wallet section.

**Review by iurimatias — APPROVED** [review]
checked in both onboarding and wallet

**Pull request merged**